src/main/resources/templates/authenticator/criipto/authenticate/authorize.vm

```diff
@@ -48,7 +48,7 @@
     #if ($iframeHeight == 0 && $iframeWidth == 0)
         ## The page will be displayed in a hidden iframe, so we'll show a spinner and some prompts
         
-        <p class="h4 mb4 semibold center" id="poll_message">#message("${_templatePrefix}.view.title")</p>
+        <p class="h4 mb4 semibold center" id="poll_message">#message("${_templatePrefix}.view.title") $!country BankID</p>
```

Comment:
`NemID BankID` doesn't make sense. This should be either:
* `Swedish BankID`
* `Norwegian BankID`
* `NemID`
Also, Swedish and Norwegian should be localized.

Reply:
okay

Reply:
You probably want to create three different message keys:
- view.title.norwegian
- view.title.danish
- view.title.swedish
Then, handle the localization and differences in those 3 keys.

Reply:
yes, would do something like this

Reply:
done